Woche2/scrabble/scrabble.c

```diff
@@ -0,0 +1,44 @@
+#include <ctype.h>
+#include <cs50.h>
+#include <stdio.h>
+#include <string.h>
+
+int POINTS[] = {1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3, 1, 1, 3, 10, 1, 1, 1, 1, 4, 4, 8, 4, 10};
+
+int compute_score(string word);
+
+int main(void) {
+    
+    string word1 = get_string("Player 1: ");
+    string word2 = get_string("Player 2: ");
+
+    int score1 = compute_score(word1);
+    int score2 = compute_score(word2);
+
+    bool Score1IsHigher = score1 > score2;

```

Comment:
```suggestion
    bool score1_is_higher = score1 > score2;

```

Variablen in [snake_case](https://en.wikipedia.org/wiki/Snake_case) halten

Reply:
In den anderen Dateien auch